packages/dd-trace/test/profiling/exporters/agent.spec.js

```diff
@@ -306,86 +303,88 @@ describe('exporters/agent', function () {
     })
   })
 
-  describeOnUnix('using UDS', () => {
-    let listener
-
-    beforeEach(done => {
-      url = `${path.join(os.tmpdir(), `dd-trace-profiler-test-${Date.now()}`)}.sock`
+  if (os.platform() !== 'win32') {
```

Comment:
A lot of the changes in this file look like they should be done in a separate PR. Any reason why you included them here?

Reply:
There is only 1 small change, which is replacing the `describeOnUnix` function with a condition, because `describe.skip` doesn't exist in node-tap.